src/app/api/auth/login/google/route.ts

```
@@ -1,9 +1,12 @@
 import 'server-only'
-import type { User } from '@/hooks/useUserStore'
 import { createSessionCookie } from '@/app/api/session'
-import { OAuth2Client } from 'google-auth-library'
-import { NextResponse, type NextRequest } from 'next/server'
+import { OAuth2Client, TokenPayload } from 'google-auth-library'
```

ESLint

[error] 'TokenPayload' is defined but never used.

(@typescript-eslint/no-unused-vars)